lib/puppet/provider/network_config/redhat.rb

```diff
@@ -171,11 +171,10 @@ def self.munge(pairs)
   end
 
   def self.format_file(filename, providers)
-    if providers.length == 0
-      return ''
-    elsif providers.length > 1
-      fail Puppet::DevError, "Unable to support multiple interfaces [#{providers.map(&:name).join(',')}] in a single file #{filename}"
-    end
+    return '' if providers.empty?
+    raise Puppet::DevError,\
+          "Unable to support multiple interfaces [#{providers.map(&:name).join(',')}] in a single file #{filename}"\
+          if providers.length > 1
```

Comment:
the first one isn't because of the comma, but the second one is not optional. I'm impartial to having either `,\` or just `,`